ToDo-Garden/ToDo-Garden/ManageGroupScene/Package.swift

```diff
@@ -0,0 +1,42 @@
+// swift-tools-version: 5.10
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+  name: "ManageGroupScene",
+  platforms: [.iOS(.v15)],
+  products: [
+    .library(
+      name: "ManageGroupSceneAPI",
+      targets: ["ManageGroupSceneAPI"]),
```

Comment:
p3;
```suggestion
    .library(
      name: "ManageGroupSceneAPI",
      targets: ["ManageGroupSceneAPI"]
     ),
```

마지막 괄호에서 엔터하나를 넣어주는게 [컨벤션](https://www.notion.so/2976668c0f8e49f09611c3ba74eb929f?pvs=4#8f1270bb860d4de499dae566a100c1cf)에 맞을 것 같습니다!!

Reply:
수정 완료 
> [3b2a281](https://github.com/ToDo-Garden/ToDo-Garden/pull/264/commits/3b2a28107f65143a53bb59b18981878476295eca)